myapp/docker-compose.yml

```diff
@@ -13,14 +13,26 @@ services:
 
   api:
     build: .
-    command: bash -c "bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    command: bash -c "bundle install && yarn install && bundle exec rails webpacker:install && bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
[Memo]
I hope there are other ways of doing this but at least it work and can prevent lots of time lost for new comers.

Reason: running yarn install and bundle install every time when restarting the container takes times. For long term usage, we didn't change gemfile/yarn too often.

Reply:
true maybe adding these to `Dockerfile` would be better

Reply:
I have tried with two ways, in both ways it takes almost same time (4-5 minutes) to build.

Reply:
well then, let's keep it that way!